src/components/charts/hubblePlot/index.jsx

```diff
@@ -438,25 +539,30 @@ class HubblePlot extends React.Component {
     } else if (!preSelected && createUserHubblePlot) {
       $hubblePlot.on('click', () => {
         const pointData = d3Select(d3Event.target).datum();
+        const isHtmlClick = d3Event.path.some(
```

Comment:
check to see if this click bubble up from a `foreignObject`, if so it should be ignored.